tgui/packages/tgui/interfaces/ChemFilter.tsx

```diff
@@ -11,37 +11,30 @@ type Data = {
 type Props = {
   title: string;
   list: string[];
-  reagentName: string;
-  onReagentInput: (str: string) => void;
+  buttonTitle: string;
```

Comment:
I don't think custom title is necessary at all, since it will always be adding reagents, regardless of the context.

```suggestion
```